packages/stanislav-sequence/src/package.ts

```diff
@@ -0,0 +1,333 @@
+  // your code goes here
+  return new DG.Widget(ui.divText(dna_nucleotide(nucleotide)));;
+}
+
+//name: executeFunction
+//tags: panel, widgets
+//input: string sequence 
+//input: string subsequence    
+//output: int result 
+export async function executeCountSubsequence(sequence: string, subsequence: string): Promise<number> {
+  return await grok.functions.call('StanislavSequence:CountSubsequencePython', { sequence, subsequence });
+}
+
+//name: getOrders
+//input: string country 
+//output: dataframe df
+export async function getOrders(country: string): Promise<DG.DataFrame> {
+  return await grok.data.query(`StanislavSequence:OrdersByCountry`, { country: country });
+}
+
+//input: string filepath
+//output: dataframe df
+export async function openTable1(filepath: string): Promise<DG.DataFrame> {
+  const df = await grok.data.getDemoTable(filepath);
+  grok.shell.addTableView(df);
+  return df;
+}
+
+//input: string filepath
+//output: dataframe df
+export async function openTable2(filepath: string): Promise<DG.DataFrame> {
+  const df = await grok.data.files.openTable(`System:/${filepath}`);
+  grok.shell.addTableView(df);
+  return df;
+}
+
+//input: string filepath
+//output: dataframe df
+export async function openTable3(filepath: string): Promise<DG.DataFrame> {
+  const df = (await (grok.functions.eval(`OpenServerFile("System:DemoFiles/${filepath}")`)))[0];
+  grok.shell.addTableView(df);
+  return df;
+}
+
+//input: string filepath
+//output: dataframe df
+export async function openTable4(filepath: string): Promise<DG.DataFrame> {
+  const df = await grok.data.loadTable(`https://dev.datagrok.ai/${filepath}`);
+  grok.shell.addTableView(df);
+  return df;
+}
+
+//name: Add Tables
+//output: string t
+export async function addTables(): Promise<String> {
+  const files = await _package.files.list('', true);
+
+  const csvFiles = files.filter((f) => f.extension === 'csv');
+  let t = "";
+  // Load every table and add a view for it
+  for (const file of csvFiles) {
+    const df = await _package.files.readCsv(file.fileName);
+    t = t + file + ' ';
+    grok.shell.addTableView(df);
+    // const df = await grok.data.loadTable(`${_package.webRoot}${file.path}`);
+    // const df = await grok.data.files.openTable(`System:AppData/${_package.name}/${file.fileName}`);
+  }
+  return t;
+}
+
+//name: fuzzyJoin
+//input: dataframe df1
+//input: dataframe df2
+//input: column col
+//input: int N 
+//output: dataframe result
+export function fuzzyJoin(df1: DG.DataFrame, df2: DG.DataFrame, col: DG.Column, N: number) {
+  const col1 = df1.columns.byName(col.name).toList();
+  const col2 = df2.columns.byName(col.name).toList();
+  const col1Length = col1.length;
+
+  let resultDF = df1.append(df2);
+  resultDF.columns.addNew("Subsequence Count", "int")
+
+  let i = 0;
+  for (let item of resultDF.rows) {
+    if (i < col1Length) {
+      item["Subsequence Count"] = getCountOfSubsequencesInColumn(item[col.name], N, col2)
```

Comment:
you could use just column.set(idx, value) for this, but this looks ok as well I guess